library/pk_wrap.c

```diff
@@ -1164,15 +1116,14 @@ static int ecdsa_sign_wrap( void *ctx_arg, mbedtls_md_type_t md_alg,
     psa_key_attributes_t attributes = PSA_KEY_ATTRIBUTES_INIT;
     mbedtls_svc_key_id_t key_id = MBEDTLS_SVC_KEY_ID_INIT;
     psa_status_t status;
-    mbedtls_pk_context key;
-    size_t key_len;
-    unsigned char buf[MBEDTLS_PK_ECP_PRV_DER_MAX_BYTES];
-    unsigned char *p;
+    unsigned char buf[PSA_KEY_EXPORT_ECC_KEY_PAIR_MAX_SIZE(
+            PSA_VENDOR_ECC_MAX_CURVE_BITS )];
```

Comment:
In case the above comment makes sense to you, what about creating a define to `MBEDTLS_PSA_MAX_EC_PUBKEY_LENGTH` for this purpose?
Perhaps `PSA_EXPORT_PUBLIC_KEY_MAX_SIZE` is already OK for this?